src/Microsoft.Graph.Core/Requests/BaseRequest.cs

```diff
@@ -296,7 +309,7 @@ private string InitializeUrl(string requestUrl)
                 throw new ServiceException(
                     new Error
                     {
-                        Code = GraphErrorCode.InvalidRequest.ToString(),
+                        Code = "invalidRequest",
```

Comment:
Why the move to hardcoded strings? I'd advocate that these should be in a constants file somewhere.